src/components/chips/chips.scss

```diff
@@ -48,13 +56,46 @@ $chip-wrap-padding: 0.8rem !default;
     display: inline-block;
     line-height: $chip-height;
 
+    input {
+      border:0;
+      height: $chip-height;
+      line-height: $chip-height;
+      padding: 0;
+
+      &:focus {
+        outline:none;
+      }
+    }
+
+    md-autocomplete, md-autocomplete-wrap {
+      height: $chip-height;
+    }
+
+    md-autocomplete {
+      box-shadow: none;
+
+      /* Override the input to the chip height */
+      input {
+        height: $chip-height;
+        line-height: $chip-height;
+        padding: 0;
+        position: relative;
+      }
+
+      button {
+        md-icon {
+          top: 1px; /* Reposition the (x) button in the autocomplete */
+        }
+      }
+    }
+
     &:not(:first-child) {
       margin: $chip-margin;
     }
-  }
 
-  .md-chip-input {
-    background-color:transparent;
-    border-width: 0;
+    input {
```

Comment:
hmm. I will limit the style to text and number for now. We can add other input types as needed